cf-agent/verify_exec.c

```diff
@@ -212,7 +212,8 @@ static ActionResult RepairExec(EvalContext *ctx, const Attributes *a,
     char eventname[CF_BUFSIZE];
     char cmdline[CF_BUFSIZE];
     char comm[20];
-    int outsourced, count = 0;
+    bool outsourced;
```

Comment:
I'd rename the variable to make more sense. `outsourced` sounds to me like it should be `true` in the parent process. What about something simple like `do_work_here`?

Reply:
Done